src/common/sr_utils.c

```diff
@@ -20,6 +20,15 @@
  * limitations under the License.
  */
 
+#include "sr_constants.h"
+#ifdef HAVE_LINUX_OFD_LOCK
+#define _GNU_SOURCE
+#else
+//! @cond doxygen_suppress
+#define __USE_XOPEN
```

Comment:
I suppose `_GNU_SOURCE` is required by the OFD locks but `__USE_XOPEN` is always required by `strptime()`, why do you not define it if OFD locks are used? That is why the tests failed. If it causes some problems, you can try defining `_XOPEN_SOURCE` instead, that is what `man` pages say one should do.

Reply:
yes, agree, when I view the headers it seems to me that _GNU_SOURCE will imply __USE_XOPEN, but that of course might be only true for my version of the compiler.  I will update to #define _XOPEN_SOURCE